package.json

```diff
@@ -128,6 +128,7 @@
     "rollup-plugin-terser": "^5.1.2",
     "start-server-and-test": "^1.10.6",
     "testcafe": "^1.8.4",
+    "testcafe-browser-provider-browserstack": "^1.13.0-alpha.1",
     "testcafe-browser-provider-saucelabs": "^1.7.0",
```

Comment:
Can we remove the saucelabs dependency?

Reply:
And are we still using `karma-sauce-launcher`? Sorry, how all our tests run on CI is still vague to me.

Reply:
Hey @Aaronius, 

- `testcafe-browser-provider-saucelabs` Can go since we are test Browserstack now.
- `karma-sauce-launcher` is being used to run our unit tests on sauce labs. Surprisingly, that bridge to SL has never failed; the one Travis provides to run the E2E tests does on a regular basis. At some point we need to investigate why; and if Browserstack does not fail in the next few weeks, maybe we can use it for our unit tests as well and consolidate it.